docassemble/AssemblyLine/al_document.py

```diff
@@ -1393,25 +1419,38 @@ def ocr_pages(self):
 
     def as_pdf(
         self,
-        prefix="",
```

Comment:
Agree that the new term is better, but I do worry that the old one might already be in use. Maybe add an alias, or do a quick GitHub search?